src/components/landing/SliderContainer.tsx

```diff
@@ -0,0 +1,477 @@
+
+      <StSixthSlider ref={sixthSlideRef}>
+        <div className="headline-wrapper">
+          <h1>아나운서 준비생들에게 이미 검증된 스피치 학습법 </h1>
+          <h1>아나운서 쉐도잉으로</h1>
+          <h1>스피치 자신감을 키워보세요!</h1>
+        </div>
+        <Link href="/home">
+          <a className="start-button">딜리버블 시작하기</a>
+        </Link>
+        <StContact>
+          <p className="contact">Contact Us</p>
+          <p className="email">DeliverBle@gmail.com</p>
+        </StContact>
+      </StSixthSlider>
+    </StSliderContainer>
+  );
+}
+
+export default SliderContainer;
+
+const StContentContainer = styled.section`
+  display: flex;
+
+  &.step1-content {
+    margin-top: 23.6rem;
+    margin-bottom: 21.9rem;
+  }
+
+  &.step2-content {
+    margin-top: 36.1rem;
+    margin-bottom: 36rem;
+  }
+  &.step3-content {
+    margin-top: 34.2rem;
+    margin-bottom: 34.1rem;
+  }
+`;
+
+const Sth4 = styled.section`
+  & > h4 {
+    color: ${COLOR.GRAY_60};
+    ${FONT_STYLES.M_24_HEADLINE};
+  }
+  margin-top: 3.2rem;
+`;
+
+const StTextWrapper = styled.section`
+  & > h2 {
+    color: ${COLOR.MAIN_BLUE};
+    ${FONT_STYLES.B_100_LANDING};
+    font-family: 'Dongle';
+  }
+
+  & > h3 {
+    color: ${COLOR.BLACK};
+    ${FONT_STYLES.SB_36_HEADLINE};
+  }
+`;
+
+const StSliderContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+
+  .first-slide {
+    width: 100%;
+    height: 100vh;
+    min-height: 108rem;
+
+    background-image: url(${imgLandingBubble.src}), url(${imgLandingBgFirst.src});
+    background-position: -60px 96px, 0px 0px;
+    background-repeat: no-repeat;
+    background-size: 1906.64px 655px, cover;
+  }
+`;
+
+const StFirstSlider = styled.section`
+  width: 100%;
+  height: 100%;
+
+  .headline-wrapper {
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-start;
+
+    color: ${COLOR.BLACK};
+    ${FONT_STYLES.M_44_HEADLINE};
+
+    padding-top: 41.5rem;
+    margin-left: 16rem;
+  }
+
+  .start-button {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+
+    width: 26rem;
+    height: 7rem;
+
+    margin-top: 4.8rem;
+    margin-left: 16rem;
+    margin-bottom: 41.5rem;
+
+    background-color: ${COLOR.MAIN_BLUE};
+    color: ${COLOR.WHITE};
+    ${FONT_STYLES.SB_24_HEADLINE};
+
+    border-radius: 1.4rem;
+  }
+`;
+
+const StSecondSlider = styled.section`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+
+  width: 100%;
+  height: 100%;
+
+  background-image: url(${imgLandingBgSecond.src});
+  background-position: 0px 18.1rem;
+  background-repeat: no-repeat;
+  background-size: contain;
+
+  .headline {
+    margin-top: 8.8rem;
+    color: ${COLOR.BLACK};
+    font-family: 'Pretendard';
+    font-style: normal;
+    font-weight: 600;
+    font-size: 44px;
+    line-height: 150%;
+    text-align: center;
+    letter-spacing: -0.01em;
+  }
+
+  .body-card-wrapper {
+    display: flex;
+    justify-content: center;
+
+    width: 192rem;
```

Comment:
```suggestion
    width: 100%;
```
100%로 바꾸면 가운데 정렬됩니다 !!